src/coreComponents/virtualElement/ConformingVirtualElementOrder1.hpp

```diff
@@ -0,0 +1,60 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2019 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2019 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2019 Total, S.A
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All right reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file ConformingVirtualElement_1.hpp
+ */
+
+#ifndef GEOSX_VIRTUALELEMENT_CONFORMINGVIRTUALELEMENTORDER1_HPP_
+#define GEOSX_VIRTUALELEMENT_CONFORMINGVIRTUALELEMENTORDER1_HPP_
+
+#include "VirtualElementBase.hpp"
+
+namespace geosx
+{
+namespace virtualElement
+{
+class ConformingVirtualElementOrder1 final : public VirtualElementBase
+{
+public:
+
+  localIndex numQuadraturePoints;
+  localIndex numSupportPoints;
+  array1d< real64 > basisFunctionsIntegralMean;
+  array2d< real64 > basisDerivativesIntegralMean;
+  array2d< real64 > stabilizationMatrix;
```

Comment:
Thank you François! These properties should be protected for sure. They are public now for testing and because we still need to agree on suitable getters.